### DISCUSSION TITLE
How to embed the webrtc stream into an html web page?

### DISCUSSION THREAD

**Discussion by drinnird** [mrlt8/docker-wyze-bridge#949] [Q&A]
Hi, I am currently using hls.js player embedded into an html web page in order to show video from one camera via wyze-bridge. Is there an easy way to do this with the webrtc stream (since its less latency)?

**Comment by mrlt8** [comment]
I believe the easiest way would probably be to embed the webrtc using an iframe:

https://github.com/bluenviron/mediamtx#web-browsers-1

or  you could try to use `http://loclahost:5000/signaling/cam-name` which should give you the whep for the webrtc and use some JS to play it back natively.

**Reply by drinnird** [reply]
I was thinking that, however, using apple/safari web browsers enforced under corporate policy, they dont like the cross site connections. Ill check out the signaling option. That might work. Will I need to provide authentication?
